### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by syheliel** [modernizing/coca#24]
`go install github.com/phodal/cocago`
```
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

**Comment by phodal** [comment]
just give a Pull Request ?

**Comment by phodal** [comment]
fixed by #25
